notify/impl.go

```diff
@@ -541,12 +554,50 @@ func (n *PagerDuty) notifyV2(
 		n.conf.Severity = "error"
 	}
 
+	images := make([]pagerDutyImage, len(n.conf.Images))
```

Comment:
You can simplify it further by leveraging the `tmpl` helper function.

```go
msg := &pagerDutyMessage{
   ...
  Images: make([]pagerDutyImage, len(n.conf.Images)),
  Links: make([]pagerDutyLink, len(n.conf.Links)),
   ...
}

for i, img := range n.conf.Images {
    msg.Images[i] = pagerDutyImage{
        Alt: tmpl(img.Alt),
        Src: tmpl(img.Src),
        Text: tmpl(img.Text),
    }
}

// Same for n.conf.Links
```
Templating errors are caught by the `tmplErr != nil` test after.